rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -160,23 +166,25 @@ Set a value by entering text or by clicking on a checkbox, and complete your ent
 +
-The variables are typed. You can only enter or link values of the same type.  The values entered here are only start values and they can be overwritten by other values during the course of the process run.
+The variables are typed. You can only enter or link values of the same type. The values entered here are only start values and they can be overwritten by other values during the course of the process run.
 * *User Task*:
 +
 User tasks are elements of a process model, which cannot be processed fully automatically by a bot, but which require the support of a human. Specify which users are to process the user tasks by clicking on the *User Assignment or  User Group Assignment* icon. The number of assigned users and user groups is displayed in the table. You can specify both users and user groups at the same time. The user task is then displayed to each specified user until a user reserves it for processing.
 +
 Select the checkbox *Send email notification to all related users* when the task is ready for processing if you want all specified users to receive an email as soon as they are supposed to process a user task. This email contains the name of the user task, the process, and the project manager, as well as a prompt to review the pending tasks in the *My RPA* module.
 * *Execution*:
 +
-Decide whether to deploy the production configuration in an existing desktop session or in a secure session, and specify an appropriate user.
+Decide whether the productive run is to be carried out in an existing console or RDP session or in a secure session, and specify an appropriate user.
 +
-For implementation in a secure session, you can link the user with a global credential variable. To do this,  click on the *Link a global activity parameter* icon.
+You can link the user with a global credential variable. To do this,  click on the *Link a credential from credential pool* icon.
 +
 Use the syntax `domain\username` instead of `username@domain.com` for domain users and `username` for local users, regardless of whether you enter the username manually or link a global credential.
 +
-Make sure the bot you want to execute the process is configured for the execution type, because you cannot save the test plan otherwise.
+Make sure the bot you want to execute the process is configured for the execution type, because you cannot save the run configuration otherwise.
```

Comment:
```suggestion
Ensure that the bot you that want to execute the process is configured for the execution type because you cannot save the run configuration otherwise.
```